kratos/geometries/hexahedra_3d_27.h

```diff
@@ -1366,6 +1374,331 @@ template<class TPointType> class Hexahedra3D27 : public Geometry<TPointType>
         return d_shape_f_values;
     }
 
+       /**
+     * returns the second order derivatives of all shape functions
+     * in given arbitrary points
+     * @param rResult a third order tensor which contains the second derivatives
+     * @param rPoint the given point the second order derivatives are calculated in
+     */
+    ShapeFunctionsSecondDerivativesType& ShapeFunctionsSecondDerivatives( ShapeFunctionsSecondDerivativesType& rResult, const CoordinatesArrayType& rPoint ) const override
+    {
+        if ( rResult.size() != this->PointsNumber() )
+        {
+            // KLUDGE: While there is a bug in ublas vector resize, I have to put this beside resizing!!
+            ShapeFunctionsGradientsType temp( this->PointsNumber() );
+            rResult.swap( temp );
+        }
```

Comment:
```suggestion
        if ( rResult.size() != this->PointsNumber() ) {
            rResult.resize(this->PointsNumber());
        }
```
I think this is no longer needed.